### PR TITLE
FAI-12277 Improve Bitbucket PR diff stats 

### PR DESCRIPTION
**Pull request by ypc-faros**

### CODE REVIEW

File: destinations/airbyte-faros-destination/src/converters/bitbucket/pull_requests_with_activities.ts [hunk 1]
```diff
@@ -85,15 +102,55 @@ export class PullRequestsWithActivities extends BitbucketConverter {
...
+          additions: diff.linesAdded,
+          deletions: diff.linesRemoved,
+        },
+      };
+    };
+
+    if (diff.old?.path) {
+      res.push(processDiffPath(diff.old.path));
+    }
+    if (diff.new?.path && diff.new.path !== diff.old?.path) {
```
Comment (matiaslcoulougian) [inline destinations/airbyte-faros-destination/src/converters/bitbucket/pull_requests_with_activities.ts:141]: If new path != old path, should we still process old path?

Comment (matiaslcoulougian) [reply destinations/airbyte-faros-destination/src/converters/bitbucket/pull_requests_with_activities.ts:141]: The only case they are both old and new defined and different from each other is when it is a file rename. In that case we would write both files with 0 lines added and 0 lines removed.

**Review by matiaslcoulougian — APPROVED** [review]
LGTM. Remove unused imports

File: destinations/airbyte-faros-destination/src/converters/bitbucket/pull_requests_with_activities.ts (hunk 1 above)
Comment (ypc-faros) [reply destinations/airbyte-faros-destination/src/converters/bitbucket/pull_requests_with_activities.ts:141]: Yeah 👍 
Added one test record to illustrate this case: https://github.com/faros-ai/airbyte-connectors/pull/1699/commits/81cfdfab33c40618769180d1030e194c9e40fa1a
It's working as intended

**Pull request merged**